extensions/identity-hub/build.gradle.kts

```
@@ -20,16 +20,22 @@ val assertj: String by project
 val edcVersion: String by project
 val edcGroup: String by project
 val jupiterVersion: String by project
+val restAssured: String by project
+val faker: String by project
 
-dependencies{
+dependencies {
     implementation("${edcGroup}:core:${edcVersion}")
     implementation("${edcGroup}:http:${edcVersion}")
```

Can this be testImplementation only ? 
```suggestion
    testImplementation("${edcGroup}:http:${edcVersion}")
```

---

Not related to this PR so can be checked separately. Just adding as a feedback.